GNUmakefile

```diff
@@ -372,7 +373,21 @@ protoc-install:
 		chmod +x $(PROTOC_ROOT)/bin/protoc ; \
 	fi
 
-proto: protoc-install $(PROTOGOFILES) $(PROTOGOBINFILES)
+.PHONY: proto
+proto: -protoc-files mog
+
+.PHONY: mog
+mog:
+	@for FULL_PKG in $(PROTO_MOG_ORDER); do \
+		PKG="$${FULL_PKG/#github.com\/hashicorp\/consul\//.\/}" ; \
+		find "$$PKG" -name '*.gen.go' -delete ; \
+		echo "mog -tags '$(GOTAGS)' -source \"$${PKG}/*.pb.go\"" ; \
+		mog -tags '$(GOTAGS)' -source "$${PKG}/*.pb.go" ; \
```

Comment:
`-ignore-package-load-errors` is not required now with the change in https://github.com/hashicorp/mog/pull/9. It uses this glob syntax to selectively load files in the package that only match the glob, and avoids loading the bonus code that we have living in those packages.

Also ensuring that we execute mog in dependency graph order cuts down on this further.